citation.cff

```diff
@@ -0,0 +1,42 @@
+cff-version: 1.0.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Zhu"
+  given-names: "Andrew"
+  orcid: "https://orcid.org/0000-0002-6664-3215"
+- family-names: "Dugan"
+  given-names: "Liam"
+  orcid: "https://orcid.org/0009-0006-5382-5235"
+- family-names: "Hwang"
+  given-names: "Alyssa"
+  orcid: "https://orcid.org/0009-0006-4827-8505"
+- family-names: "Callison-Burch"
+  given-names: "Chris"
+  orcid: "https://orcid.org/0000-0001-8196-1943"
+title: "kani"
+version: 1.0.0
+date-released: 2023-09-11
```

Comment:
do we want to bump this with the actual package releases or just have a citation/paper version?

Reply:
up to you